# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/moz.build
# Commit: 85974aa6ae0b
# Full Hash: 85974aa6ae0bf20be02a09ecfcd3aa6f27ebcb91
# Author: stransky <stransky@redhat.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1751987
# File Overlap Count: 3
# Description:
#   Bug 1751987 [Linux] Implement VAAPIUtils to get basic VA-API info r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139299
# ==============================================================================

diff -r 469777a5cb87 -r 85974aa6ae0b widget/gtk/moz.build
--- a/widget/gtk/moz.build	Wed Feb 23 10:37:35 2022 +0000
+++ b/widget/gtk/moz.build	Wed Feb 23 11:47:06 2022 +0000
@@ -92,6 +92,7 @@
         "MozContainerWayland.cpp",
         "nsClipboardWayland.cpp",
         "nsWaylandDisplay.cpp",
+        "VAAPIUtils.cpp",
         "WaylandBuffer.cpp",
         "WindowSurfaceWaylandMultiBuffer.cpp",
     ]
@@ -100,8 +101,15 @@
         "DMABufSurface.h",
         "MozContainerWayland.h",
         "nsWaylandDisplay.h",
+        "VAAPIUtils.h",
         "WaylandBuffer.h",
     ]
+    LOCAL_INCLUDES += [
+        "/media/mozva",
+    ]
+    USE_LIBS += [
+      "mozva",
+    ]
 
 if CONFIG["MOZ_X11"]:
     UNIFIED_SOURCES += [
